src/content/changelog/4-65-0-2020-05-18.md

```diff
@@ -0,0 +1,21 @@
+---
+title: '4.65.0: 2020-05-18'
+date: 2020-05-18T04:00:00.000Z
+version: 4.65.0
+changelog:
+  - API
+---
+
+### Added
+
+- The `managed_issue` field was added to the Open Support Ticket endpoint ([POST /support/tickets](/api/v4/support-tickets/#post)).
+    - This field designates if the Support ticket is in reference to a [Managed Service](https://www.linode.com/products/managed/).
+
```

Comment:
Can we add a note about how you can't have the other ids present if this field is passed in?